### PR TITLE
Dailydouble

### PR DESCRIPTION
**Pull request by K-Rye**
- Built dailydouble class in DailyDouble.js file 
- created validate wager function to check if a users wager is valid based on the parameters stated
- Created testing suite in DailyDlouble-test.js to test wager validation. 
- May need to revisit to add functionality.

 

@brittanystoroz @thatpamiam

### CODE REVIEW

File: js/DailyDouble.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  constructor(clue) {
+    super(clue)
+  }
+
+  validateWager(wager, currentPlayer) {
+    if (isNaN(wager) || wager > currentPlayer.points || wager < 5) {
+      return false;
+    } else {
+      return true;
+    };
```
Comment (chris023) [inline js/DailyDouble.js:15]: I'm not sure that leaving semicolon's like this at the end of block scope curly braces is correct styling -- lets run these through a linter and remerge

File: js/Game.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Game  {
```
Comment (chris023) [inline js/Game.js:1]: We shouldn't be tracking changes to this file in this branch -- when using git add, just add the files that are relevant to this branch.

File: js/Gameboard.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+class Gameboard {
```
Comment (chris023) [inline js/Gameboard.js:1]: See previous comment about which files to commit

File: js/Player.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+class Player {
```
Comment (chris023) [inline js/Player.js:1]: See previous comment about which files to commit

File: js/Question.js [hunk 5]
```diff
@@ -6,9 +6,15 @@ class Question {
     this.pointValue = clue.pointValue
```
Comment (chris023) [inline js/Question.js:6]: See previous comment about which files to commit

File: test/DailyDouble-test.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+describe('DailyDouble', function() {
+  let dailydouble;
+  let player;
+  let question;
+  let clue = {
+      question: 'Scorecard Report" & "Peter Jacobsen Plugged In" are seen on the sports channel devoted to this',
+      answer: 'golf',
+      pointValue: 400,
+      categoryId: 10
+      }
```
Comment (chris023) [inline test/DailyDouble-test.js:16]: Lint this file

File: test/Game-test.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const chai = require('chai');
```
Comment (chris023) [inline test/Game-test.js:1]: Should only be tracking the test file for daily double

**Review by chris023 — APPROVED** [review]

File: js/DailyDouble.js (hunk 1 above)
Comment (thatPamIAm) [reply js/DailyDouble.js:15]: Right on that count!

`{` and `}` are meant to begin and close a group of statements so semi-colons shouldn't be used.

**Comment by thatPamIAm** [comment]
@K-Rye Nice job with being descriptive in your PR and @chris023 good job making line-by-line comments over the files that were submitted!

**Pull request merged**